.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,122 @@
+  - type: checkboxes
+    id: version
+    attributes:
+      label: Are you using the latest Dosbox-Staging Version?
+      description: |
+        What version are you using? Check the [latest release](https://github.com/dosbox-staging/dosbox-staging/releases)
+        Run `dosbox --version`
+        Type `ver` command inside DOSBox shell; If the version is reported as `git` (meaning you compiled it yourself), use `git describe` command.
+        Provide 'git' version or 'other' in description
+      options:
+        - label: I have checked releases and am using the latest release.
+          required: true
+  - type: input
+    id: verinfo
+    attributes:
+      label: Different version than latest?
+      description: Alpha, Git, or Branch?
+      placeholder: "Ex. 0.78.0-alpha, git verion, branch testing."
+    validations:
+      required: false
+  - type: dropdown
+    id: OS
+    attributes:
+      label: What Operating System are you using?
+      multiple: false
+      options:
+        - Windows 11
+        - Windows 10
+        - Windows 8
+        - Windows 7
+        - Windows Other
+        - Mac OS
+        - Linux x86
+        - Linux x86_64
+        - Linux ARM (Raspberry Pi)
+        - Other
+    validations:
+      required: false
+  - type: input
+    id: other
+    attributes:
+      label: If Other OS, please describe
+      description: Other details
+      placeholder: "Windows, Mac OSX version, Debian, Ubuntu, Arch, etc."
+    validations:
+      required: false
+  - type: input
+    id: hardware
+    attributes:
+      label: Relevant hardware info
+      description: Hardware
+      placeholder: "CPU, GPU, device brand/model: e.g. Raspberry Pi3B+, Nvidia, etc"
+    validations:
+      required: false
+  - type: checkboxes
+    id: checked
+    attributes:
+      label: Have you checked that no other similar issue already exists?
+      description: Searched issues before creating report?
+      options:
+        - label: I have searched and not found similar issues.
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: A clear and concise description of what the bug is.
+      description: Describe what happens, what software were you running? _Include screenshot if possible_
+      placeholder: "How & When does this occur?"
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce the behaviour.
+      description: How can we reproduce this?
+      value: |
+        Explain how to reproduce
+        1. 
+        2.
+        3.
+    validations:
+      required: false
+  - type: textarea
+    id: config_file
+    attributes:
+      label: Your configuration
+      description: |
+        Share the config file(s) you've been using to run the program. (`dosbox-staging.conf`) 
+        Please avoid pasting the full config, _use attachments or links_ in a [Gist](https://gist.github.com/)
```

Comment:
Code blocks make the config easier to read even if the config is long. Since the config section now produce code blocks with the render key we should don't need this note anymore